.github/workflows/release.yaml

```diff
@@ -55,10 +62,67 @@ jobs:
           # This expects you to have a script called release which does a build for your packages and calls changeset publish
           publish: yarn release
           version: yarn version-and-reinstall
+  
+  # release gives an array of published packages as jsob objects, we need an array of strings for installing
+  prepare:
+    name: Format Output for Install
+    if: needs.release.outputs.published
+    needs: release 
+    runs-on: ubuntu-latest
+    container:
+      image: node:20-bullseye
+    outputs:  
+      result: "${{ steps.map.outputs.output }}"
+    steps:              
+      - name: Install jq
+        if: needs.release.outputs.published
+        uses: dcarbone/install-jq-action@v3.0.1
+      - name: Format Published Packages Array
+        if: needs.release.outputs.published
+        uses: cloudposse/github-action-jq@main
+        id: map
+        with:
+          compact: true
+          raw-output: true
+          input: '${{ needs.release.outputs.publishedPackages }}'
+          script: |-
+            map("\(.name)@\(.version)")
+
+  # last line of defense to ensure that the packages were published correctly
+  install-released-packages:
+    name: Install Released Packages
+    needs: [prepare, release]
+    if: ${{ needs.release.outputs.published }} == 'true' 
+    runs-on: ubuntu-latest
+    container:
+      image: node:20-bullseye
+    strategy:
+      fail-fast: false
+      max-parallel: 12
+      matrix:
+        package: ${{fromJson(needs.prepare.outputs.result)}}
+    steps:
+      - name: Install @celo/celocli dependencies
+        if: contains(matrix.package, '@celo/celocli')
+        run: |
+          apt update
+          apt install -y libusb-1.0-0-dev libudev-dev
+          npm install node-gyp --global
+
+      - name: Installing ${{ matrix.package }} package
+        run: npm install ${{ matrix.package }} --global
+      - name: Ensure sample of celocli commands run
```

Comment:
this is awesome 👍

Reply:
we can add more too but i figure a sample of view commands will catch any wierdness